docs/reference/field-types/checkboxes.md

```diff
@@ -46,6 +46,8 @@ genres: {
 |`if` | Object | `{}` | Conditions to meet before the field is active. [See the guide for details.](/guide/conditional-fields) |
 |`required` | Boolean | `false` | If `true`, the field is mandatory |
 |`readOnly` | Boolean | `false` | If `true`, prevents the user from editing the field value |
+| `style` | String | n/a | If set to 'combo', it adds a selection choices box and converts the choices to a dropdown list |
+| 'all' | Object | n/a | If `style` is set to 'combo' it adds a choice of all checkboxes and accepts an object with a key of `label` and value of the label for the choice |
```

Comment:
I find this language confusing. How about: "if style is also set to combo, this adds an "All" choice to select every option."

Reply:
Changed - hopefully clearer.